### PR TITLE
[Docs] More FAQs

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Notes for Reviewers**

This PR fixes #

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by alphaX86 — COMMENTED** [review]
My opinions... Rest look fine to me

File: site/src/assets/data/faq/index.js [hunk 1]
```diff
@@ -1,14 +1,70 @@
...
+                "",
+            ],
+        },
+*/
+
+        {
+            question: "Why do I need to sign-up/register to use the Cloud Native Playground?",
+            category: "Meshery Playground",
+            answer: [
+                "Sign-up to receive a user account, entitled with perpetually free access, because the Playground is not a crypto mine. The Meshery Playground is connected to live Kubernetes cluster(s) and allows users full-control over those clusters. ",
```
Comment (AugustasV) [inline site/src/assets/data/faq/index.js:22]: TLDR, let's make point straight away.
This account will give you free and perpetual access to the Playground. The reason for signing up is that the Playground is not a cryptocurrency mining platform. It is actually connected to live Kubernetes clusters, and by signing up, you gain full control over those clusters.

File: site/src/assets/data/faq/index.js [hunk 2]
```diff
@@ -1,14 +1,70 @@
...
             answer: [
                 "The Cloud Native Playground (aka Meshery Playground) is a managed instance of Meshery that offers a sandbox environment in which half of MeshMap functionality is enabled (Designer mode) and the other half of MeshMap functionality is disabled (Visualizer mode).",
-                "The sandbox environment is not connected to an active Kuberentes cluster, and as such, specific actions within MeshMap Designer are also disabled. Meshery and MeshMap are feature-rich, sophisticated management applications for cloud native infrastructure. To access their full set of capabilities, simply deploy your own copy of Meshery into the environment of your choosing.",
+                "The sandbox environment is not connected to an active Kuberentes cluster, and as such, depending upon which Meshery Playgrouns environment you use specific actions within MeshMap Designer are also disabled. Meshery and MeshMap are feature-rich, sophisticated management applications for cloud native infrastructure. To access their full set of capabilities, simply deploy your own copy of Meshery into the environment of your choosing.",
```
Comment (AugustasV) [inline site/src/assets/data/faq/index.js:47]: Before it was mentioned that `The Meshery Playground is connected to live Kubernetes cluster(s)` and right now it's said that it's sandbox. Could that be written in more clear way?

File: site/src/assets/data/faq/index.js [hunk 3]
```diff
@@ -1,14 +1,70 @@
...
+                "Yes, Meshery can be deployed on-premises with a single command to download, install, and run your own instance of Meshery in your environment.",
+                "See all supported platforms to deploy your own Meshery instance at https://meshery.io",
             ],
         },
     ]
 }
 
+
+ 
 export default data;
```
Comment (AugustasV) [inline site/src/assets/data/faq/index.js:70]: new line missing

**Review by AugustasV — COMMENTED** [review]
Looks good! Some insights.

File: site/src/assets/data/faq/index.js (hunk 1 above)
Comment (leecalcote) [reply site/src/assets/data/faq/index.js:22]: @AugustasV, you're very much right about this suggestion. The other phrasing is indirect and leaves much to the imagination of the reader.

[the remark this answers is not in the document]
Comment (leecalcote) [reply site/src/assets/data/faq/index.js:22]: :)

File: site/src/assets/data/faq/index.js [hunk 4]
```diff
@@ -1,14 +1,70 @@
...
+            answer: [
+                "Sign-up to receive a user account, entitled with perpetually free access, because the Playground is not a crypto mine. The Meshery Playground is connected to live Kubernetes cluster(s) and allows users full-control over those clusters. ",
+                "Requiring users to sign up  and register a user account keeps the Playground a safe and healthy for all to enjoy. Without requiring users to sign-in the Meshery Playground would allow anonymous access for anyone and everyone to potentially perform nefarious acts. ",
+            ],
+        },
+        {
+            question: "Will I lose my work in the Cloud Native Playground?",
+            category: "Meshery Playground",
+            answer: [
+                "No, all of the designs that you have created in the Meshery Playground are saved to your user account and will be available to you each time that you sign into your account. Your designs will be available in the Meshery Playground and any your other Meshery deployments, too.",
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply site/src/assets/data/faq/index.js:30]: I have split this question into two questions to help drive clarity.

File: site/src/assets/data/faq/index.js (hunk 1 above)
Comment (leecalcote) [reply site/src/assets/data/faq/index.js:22]: Rephrased.

File: site/src/assets/data/faq/index.js (hunk 2 above)
Comment (leecalcote) [reply site/src/assets/data/faq/index.js:47]: Great catch. The first version of the Playground was disconnected from any live Kubernetes cluster. This new version _is_ connected. I'm updating this...

File: site/src/assets/data/faq/index.js (hunk 3 above)
Comment (leecalcote) [reply site/src/assets/data/faq/index.js:70]: Added new line

**Comment by leecalcote** [comment]
Thank you, gentlemen. I think I incorporate all the feedback.

**Pull request merged**